### PR TITLE
feat: support dynamic styling for prism

### PR DESCRIPTION
**Pull request by kevinslin**
Prism is used for code highlighting. support dynamic display of prism

### CODE REVIEW

File: packages/dendron-next-server/pages/vscode/note-preview.tsx [hunk 1]
```diff
@@ -100,12 +117,13 @@ function Note({ engine, ide }: DendronProps) {
...
   const dispatch = engineHooks.useEngineAppDispatch();
-  const { switcher, themes, currentTheme, status } = useThemeSwitcher();
+  const { currentTheme } = useThemeSwitcher();
   const themeType = getThemeType(currentTheme);
+  logger.info({ctx, currentTheme, themeType})
   useMermaid({ config: engine.config, themeType });
 
   const { noteActive } = ide;
-  const { id: noteId = "links.target", contentHash } =
+  const { id: noteId = "9eae08fb-5e3f-4a7e-a989-3f206825d490", contentHash } =
```
Comment (hikchoi) [inline packages/dendron-next-server/pages/vscode/note-preview.tsx:126]: is this a special note?

Comment (kevinslin) [reply packages/dendron-next-server/pages/vscode/note-preview.tsx:126]: yeah, its just used for testinig

File: bootstrap/scripts/createBuild.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ lerna version $version --no-git-tag-version --no-push
 
 git add .
 git commit -m "integ: publish $version"
-git push --set-upstream origin integ-publish --force
+git push --set-upstream origin integ-publish --force --no-verfy
```
Comment (hikchoi) [inline bootstrap/scripts/createBuild.sh:11]:
```suggestion
git push --set-upstream origin integ-publish --force --no-verify
```

File: packages/dendron-next-server/pages/vscode/note-preview.tsx (hunk 1 above)
Comment (hikchoi) [reply packages/dendron-next-server/pages/vscode/note-preview.tsx:126]: got it.

**Review by hikchoi — APPROVED** [review]
Looks good other than the typo

**Pull request closed without merging**